build.sbt

```diff
@@ -49,6 +53,8 @@ libraryDependencies ++= Seq(
   "eus.ixa" % "ixa-pipe-nerc" % "1.6.1"
 )
 
+libraryDependencies += "nz.ac.waikato.cms.weka" % "optics_dbScan" % "1.0.4"
```

Comment:
Yeah, I was trying this to compare to dbscan on spark - both of which may not be ready for prime time for our use.